x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/execution/search/Querier.java

```diff
@@ -150,6 +151,30 @@ public static SearchRequest prepareRequest(Client client, SearchSourceBuilder so
                         includeFrozen ? IndexResolver.FIELD_CAPS_FROZEN_INDICES_OPTIONS : IndexResolver.FIELD_CAPS_INDICES_OPTIONS)
                 .request();
     }
+    
+    protected static void logSearchResponse(SearchResponse response, Logger logger) {
+        List<Aggregation> aggs = Collections.emptyList();
+        if (response.getAggregations() != null) {
```

Comment:
Nit: wondering if expanding this branch over the `for` loop wouldn't also simplify (i.e. remove) the in-loop conditional, by allowing to always append `", "` and then strip last two chars outside the loop.

Reply:
Thanks. I do have a slight preference towards the current approach. I'll merge it as is.